jobs/tests.py

```diff
@@ -90,3 +88,43 @@ def test_set_expiration_date_twice(self):
         second_date = self.job.expiration_date
         self.assertTrue(self.job.expiration_date, "Job has no expiration date.")
         self.assertTrue(second_date.second > first_date.second)
+
+
+class MeetupModelTests(TestCase):
+    """Tests for custom methods of Meetup model."""
+
```

Comment:
Model mommy is an easy way to mage fixtures to use for testing. Fixtures
are fixed and reusable data that you load for testing, rather than creating
actual objects every time you test. Mommy is nice because it lets you make
objects with foreign keys, multiple instances etc without having to define
fields you're not specifically testing
On 12 Feb 2015 16:24, "Marysia" <notifications@github.com> wrote:

> In jobs/tests.py
> <https://github.com/djangogirlsjobs/djangogirls/pull/7#discussion_r24593822>
> :
>
> > @@ -90,3 +88,43 @@ def test_set_expiration_date_twice(self):
> >          second_date = self.job.expiration_date
> >          self.assertTrue(self.job.expiration_date, "Job has no expiration date.")
> >          self.assertTrue(second_date.second > first_date.second)
> > +
> > +
> > +class MeetupModelTests(TestCase):
> > +    """Tests for custom methods of Meetup model."""
> > +
>
> I wasn't sure what is the advantage of using model_mommy...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/djangogirlsjobs/djangogirls/pull/7/files#r24593822>.
>